src/System.Memory/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Boolean.cs

```diff
@@ -10,7 +10,7 @@ public static partial class Utf8Formatter
         /// Formats a Boolean as a UTF8 string.
         /// </summary>
         /// <param name="value">Value to format</param>
-        /// <param name="buffer">Buffer to receive UTF8 string</param>
+        /// <param name="buffer">Buffer to write the UTF8-formatted value to</param>
         /// <param name="bytesWritten">Receives the length of the formatted text in bytes</param>
```

Comment:
The same nit about using "receive" in the xml comments applies to other parameters, like bytesWritten.